pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -277,30 +289,25 @@ metadata:
   name: jiva-volume-isvalidversion-default
 spec:
   meta: |
-    id: is070jivavolume
-    runNamespace: {{.Volume.runNamespace}}
+    id: is082jivavolume
+    runNamespace: {{ .Volume.runNamespace }}
     apiVersion: v1
     kind: Service
     action: list
     options: |-
-      labelSelector: vsm={{ .Volume.owner }}
+      labelSelector: openebs.io/controller-service=jiva-controller-svc,openebs.io/version=0.8.2
   post: |
-    {{- jsonpath .JsonResult "{.items[*].metadata.name}" | trim | saveAs "is070jivavolume.name" .TaskResult | noop -}}
-    {{- .TaskResult.is070jivavolume.name | empty | not | versionMismatchErr "is not a jiva volume of 0.7.0 version" | saveIf "is070jivavolume.versionMismatchErr" .TaskResult | noop -}}
```

Comment:
do we need to change the message to 'is a jiva volume of 0.7.0 version" @AmitKumarDas ? I'm trying to verify the old code.

Reply:
@vishnuitta @kmova @utkarshmani1997 What i understand here is, openebs allows upgrade from ?? till 0.8.2 after which a volume cannot be upgraded; since it requires movement of volume to openebs system namespace.

In such a case, upgrade should never update the existing version info at-all.

Based on some conclusions, we should be able to design these RunTasks.